pkg/plugins/runtime/k8s/metadata/annotations.go

```diff
@@ -85,6 +91,7 @@ const (
 // Annotations related to the gateway
 const (
 	IngressServiceUpstream = "ingress.kubernetes.io/service-upstream"
+	AnnotationBuiltin      = "builtin"
```

Comment:
super nitpicky, but cuddling this declaration up with `IngressServiceUpstream` makes it seem related to it

Reply:
That annotation is used by the existing gateway mode so it's somewhat related..